### PR TITLE
DOC-2241: TinyMCE 6.8.3 release notes

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2241

Changes:
* added `6.8.3-release-notes.adoc` to project; updated `changelog.adoc`, `nav.adoc` and `release-notes.adoc` for the TinyMCE 6.8.3 release.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] `modules/ROOT/nav.adoc` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/6.8.3-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+:description: Release notes for TinyMCE 6.8.3
+:keywords: releasenotes, new, changes, bugfixes
+:page-toclevels: 1
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} 6.8.3 was released for {enterpriseversion} and {cloudname} on <weekday>, <month> <DD>^<st|nd|rd|th>^, <YYYY>. These release notes provide an overview of the changes for {productname} 6.8.3, including:
```
Comment (kemister85) [inline modules/ROOT/pages/6.8.3-release-notes.adoc:12]: - [ ] Reminder to update date once confirmed.

**Review by vpyshnenko — APPROVED** [review]

File: modules/ROOT/nav.adoc [hunk 2]
```diff
@@ -405,6 +405,10 @@
 ** xref:tinymce-and-cors.adoc[Cross-Origin Resource Sharing (CORS)]
 * Release information
 ** xref:release-notes.adoc[Release notes for TinyMCE 6]
+*** TinyMCE 6.8.3
+**** xref:6.8.2-release-notes.adoc#overview[Overview]
```
Comment (FarzadHayat) [inline modules/ROOT/nav.adoc:409]:
```suggestion
**** xref:6.8.3-release-notes.adoc#overview[Overview]
```

**Review by FarzadHayat — CHANGES_REQUESTED** [review]
fix link in nav.adoc

**Review by FarzadHayat — APPROVED** [review]
LGTM ✅

File: modules/ROOT/pages/6.8.3-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+* The **Spelling Autocorrect** plugin, see: xref:autocorrect.adoc[Spelling Autocorrect plugin].
+* Deploying the **server-side** components, see: xref:introduction-to-premium-selfhosted-services.adoc[Server-side component installation].
+
+The Java server-side components have been updated to the following versions:
+
+* `ephox-hyperlinking.war`: 2.105.24
+* `ephox-image-proxy.war`: 2.106.11
+* `ephox-spelling.war`: 2.118.20
+
+NOTE: Service log level was changed from DEBUG to INFO, to reduce logs verbosity and gain more clarity.
```
Comment (EkimChau) [inline modules/ROOT/pages/6.8.3-release-notes.adoc:47]: Remove line 47 admonition

**Review by EkimChau — APPROVED** [review]

File: changelog.md [hunk 4]
```diff
@@ -6,6 +6,7 @@ The format is loosely based on [Keep a Changelog](https://keepachangelog.com/en/
 
 ### Unreleased
 
+- DOC-2241: added `6.8.3-release-notes.adoc` to project; updated `changelog.adoc`, `nav.adoc` and `release-notes.adoc` for the TinyMCE 6.8.3 release.
```
Comment (kemister85) [inline changelog.md:9]:
```suggestion
### 2024-02-08

- DOC-2241: added `6.8.3-release-notes.adoc` to project; updated `changelog.adoc`, `nav.adoc` and `release-notes.adoc` for the TinyMCE 6.8.3 release.
```

File: modules/ROOT/pages/6.8.3-release-notes.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/pages/6.8.3-release-notes.adoc:12]:
```suggestion
{productname} 6.8.3 was released for {enterpriseversion} and {cloudname} on Wednesday, February 08^th^, 2024. These release notes provide an overview of the changes for {productname} 6.8.3, including:
```

File: modules/ROOT/pages/6.8.3-release-notes.adoc [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+* The **Link Checker** plugin, see: xref:linkchecker.adoc[Link Checker plugin].
+* The **Spell Checker Pro** plugin, see: xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro plugin].
+* The **Spelling Autocorrect** plugin, see: xref:autocorrect.adoc[Spelling Autocorrect plugin].
+* Deploying the **server-side** components, see: xref:introduction-to-premium-selfhosted-services.adoc[Server-side component installation].
+
+The Java server-side components have been updated to the following versions:
+
+* `ephox-hyperlinking.war`: 2.105.24
+* `ephox-image-proxy.war`: 2.106.11
+* `ephox-spelling.war`: 2.118.20
```
Comment (kemister85) [inline modules/ROOT/pages/6.8.3-release-notes.adoc:45]:
```suggestion
* `ephox-spelling.war`: 2.118.19
```

**Pull request merged**